corehq/db/config.yml

```diff
@@ -0,0 +1,20 @@
+# HQ database configuration for multiple database. This gets mapped to Django's DATABASES dict
```

Comment:
nice, yeah that's exactly what I was imagining. I'm quite happy for you to work on this, though I think you should confirm with @dannyroberts / @fyniac or just throw into a PID bucket. 

@snopoke @benrudolph - do you all think that what @millerdev outlined makes sense and would fit your use case?